samples/minimal/CMakeLists.txt

```diff
@@ -55,14 +50,20 @@ endif()
 
 # Define the build target for the executable
 add_executable(${PROJECT_NAME} WIN32 MACOSX_BUNDLE ${SRC_FILES})
-# Link required libraries to the executable
-target_link_libraries(${PROJECT_NAME} ${wxWidgets_LIBRARIES})
+# Link executable to the wxWidgets target
+target_link_libraries(${PROJECT_NAME} wxWidgets::wxWidgets)
```

Comment:
I wonder if this target shouldn't be called `wxWidgets::all` or something like for clarity? Or is this name fixed?

Reply:
Fixed, I want to use the same name as https://cmake.org/cmake/help/latest/module/FindwxWidgets.html#imported-targets